shark-graph/src/main/java/shark/HeapObject.kt

```diff
@@ -219,6 +206,13 @@ sealed class HeapObject {
       return hprofGraph.readClassDumpRecord(objectId, indexedObject)
     }
 
+    /**
+     * Returns the name of the field declared in this class for the specified [fieldRecord].
+     */
+    fun fieldName(fieldRecord: FieldRecord): String {
```

Comment:
Let's rename this to `instanceFieldName`, to avoid confusion with static fields?